mitodl/mit-open
# API endpoints for platforms, offerers, departments

# What are the relevant tickets?
Closes #246

# Description (What does it do?)
Adds API endpoints for:
- LearningResourcePlatform
- LearningResourceOfferor
- LearningResourceDepartment

Makes the schema documentation for the above (plus topics) a bit more user friendly,

# Screenshots (if appropriate):

<img width="5" alt="Screenshot 2023-11-29 at 3 00 54 PM" src="https://github.com/mitodl/mit-open/assets/187676/bd60be0f-b645-466d-97fa-d01040202442">

<img width="965" alt="Screenshot 2023-11-29 at 3 00 58 PM" src="https://github.com/mitodl/mit-open/assets/187676/9f771a0c-0c3e-4c7a-a0d8-e4d0594f085c">


# How can this be tested?
You should be able to load the following and get expected results, logged in or not:

http://localhost:8063/api/v1/departments/
http://localhost:8063/api/v1/platforms/
http://localhost:8063/api/v1/offerers/

The schema documentation for the above and topics should be more user-friendly than the default (see screenshots above).
